r/PACKAGING.md

```diff
@@ -0,0 +1,128 @@
+  replicate most of the checks that CRAN runs, so we need them all to be passing
+  or to understand that the failures may (though won't necessarily) result in a rejection from CRAN.
+- [ ] Check [current CRAN check results](https://cran.rstudio.org/web/checks/check_results_arrow.html)
+- [ ] Ensure the contents of the README is accurate
+- [ ] Run `urlchecker::url_check()` on the R directory at the release candidate
+  commit. Ignore any errors with badges as they will be removed in the CRAN release branch.
+- [ ] [Polish NEWS](https://style.tidyverse.org/news.html#news-release) but do **not** update version numbers (this is done automatically later).
+- [ ] For major releases, prepare tweet thread highlighting new features
+
+Wait for the release candidate to be cut:
+
+- [ ] Release candidate!
+
+Make pull requests into the [autobrew](https://github.com/autobrew) and
+[rtools-packages](https://github.com/r-windows/rtools-packages) repositories
+used by the configure script on MacOS and Windows. These pull requests will
+use the release candidate as the source.
+
+- [ ] Pull request to modify 
+  [the apache-arrow autobrew formula]( https://github.com/autobrew/homebrew-core/blob/high-sierra/Formula/apache-arrow.rb) 
+  to update the release version, SHA256 checksum of the release source file, and any changes to dependencies and build steps that have changed in the
+  [copy of the formula we have of that formula in the Arrow repo](https://github.com/apache/arrow/blob/master/dev/tasks/homebrew-formulae/autobrew/apache-arrow.rb)
+- [ ] Pull request to modify
+  [the apache-arrow-static autobrew formula]( https://github.com/autobrew/homebrew-core/blob/master/Formula/apache-arrow-static.rb)
+  to update the version, SHA, and any changes to dependencies and build steps that have changed in the
+  [copy of the formula we have of that formula in the Arrow repo](https://github.com/apache/arrow/blob/master/dev/tasks/homebrew-formulae/autobrew/apache-arrow-static.rb)
+- [ ] Pull request to modify the 
+  [autobrew script](https://github.com/autobrew/scripts/blob/master/apache-arrow)
+  to include any additions made to
+  [r/tools/autobrew](https://github.com/apache/arrow/blob/master/r/tools/autobrew).
+- [ ] Pull request to modify the
+  [PKGBUILD script](https://github.com/r-windows/rtools-packages/blob/master/mingw-w64-arrow/PKGBUILD)
+  to reflect changes in
+  [ci/PKGBUILD](https://github.com/apache/arrow/blob/master/ci/scripts/PKGBUILD),
+  uncommenting the line that says "uncomment to test the rc".
+
+Prepare and check the .tar.gz that will be released to CRAN.
+
+- [ ] `git fetch upstream && git checkout release-X.X.X-rcXX && git clean -f -d`
+- [ ] Unset the `ARROW_HOME` environment variable and then run `make build` (copies Arrow C++ into tools/cpp, prunes some unnecessary components, and runs `R CMD build`)
+- [ ] `devtools::check("arrow_X.X.X.tar.gz")` locally
+- [ ] Run reverse dependency checks
```

Comment:
Why don't we do this later, after we've picked the additional commits across, just in case some weird thing (or a cheeky last minute addition that we've added to the CRAN release) has an impact?

Reply:
I thought we tried before and it was not feasible to do revdep checks in crossbow